tests/lib/ansible_helper.py

```diff
@@ -147,8 +148,18 @@ def create_inventory(self, hardware):
         with open(nodes_inv_path, 'w') as inv_file:
             yaml.dump(inv, inv_file)
 
+        logger.info('Inventory path: {}'.format(inventory_dir))
         return inventory_dir
 
+    def run_play_raw(self, playbook):
+        path = os.path.join('tests/assets/ansible', playbook)
```

Comment:
We should make this path absolute in case somebody runs tox from another dir. I'll do it as a followup